styles.css

```diff
@@ -451,37 +451,347 @@ a {
   color: #ffffff;
 }
 
-.about {
+.about__animation {
+  display: inline-block;
+}
+.resources__layer--1,
+.resources__layer--2,
+.resources__layer--3,
+.resources__layer--4 {
+  position: absolute;
+  animation-duration: .75s;
+  -moz-animation: none !important; /* Disable in Firefox because SVG transform origin bug */
+}
+.resources__layer--1,
+.resources__layer--2,
+.resources__layer--3 {
+  fill: #ffffff;
+  stroke: #e85d5d;
+  stroke-width: 40px;
+}
+.resources__layer--4 {
   fill: #e85d5d;
 }
+.resources:hover .resources__layer--1,
+.resources:hover .resources__layer--2,
+.resources:hover .resources__layer--3,
+.resources:hover .resources__layer--4 {
+  animation-timing-function: ease-in-out;
+}
+.about__animation.animated .resources__layer--1 {
+  animation-name: resources-one;
+}
+.about__animation.animated .resources__layer--2 {
+  animation-name: resources-two;
+}
+.about__animation.animated .resources__layer--3 {
+  animation-name: resources-three;
+}
+.about__animation.animated .resources__layer--4 {
+  animation-name: resources-four;
+}
+@keyframes resources-one {
+  0% {
```

Comment:
we can also do 0%, 100% {} because they are the same. do you think we should? more terse, or is it less easy to read?